src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Figure[] FIGURES = new Figure[] {
+            new Circle(new Random().nextInt(100), ColorSupplier.randomColor()),
+            new Triangle(new Random().nextInt(100), ColorSupplier.randomColor()),
+            new Square(new Random().nextInt(100), ColorSupplier.randomColor()),
+            new Rectangle(new Random().nextInt(100), new Random().nextInt(100),
+                    ColorSupplier.randomColor()),
+            new IsoscelesTrapezoid(new Random().nextInt(100), new Random().nextInt(100),
+                    new Random().nextInt(100), ColorSupplier.randomColor())};
+
+    public static Figure getRandomFigure() {
```

Comment:
What result will I get if I call the `getRandomFigure()` method twice?

Reply:
You will get two random figures. Maybe I don't understand quastion in a right way.

Reply:
I made it non static to make figuresi in array random.